### DISCUSSION TITLE
Bug report 「缺陷报告」应用商店的搜索，建议增加类型过滤条件

### DISCUSSION THREAD

**Discussion by 157221810** [linuxdeepin/developer-center#2994] [Features Request & Ideas | 特性请求 & 头脑风暴]
**问题**：在应用商店搜索应用时，没有能设置过滤条件的地方，仅就应用的关键字作为条件进行搜索，会有大量干扰结果。如，想搜索在Deepin上能使用的原生Linux版本的桌面便签程序，搜索‘便签’关键字，推荐结果是大量的Android版应用，这是我不想要的。
**建议**：可以在搜索边上增加版本选择的过滤条件，供用户选择。默认是查找所有版本的，当用户选择了特定版本，如Linux版本的应用 ，则搜索结果中去掉非Linux的Android、Wine版本的应用。

**Comment by meiyixiang** [comment]
这个功能会在20.6版本时支持，请继续支持我们

**Comment by meiyixiang** [comment]
I have documented this suggestion, in version 20.6 there will be filtering capabilities that do not apply
